Assets/MRTK/SDK/Features/UX/Scripts/Slate/FollowMeToggle.cs

```diff
@@ -117,6 +118,13 @@ public Transform AutoFollowTransformTarget
         private RadialView radialView = null;
         private Coroutine autoFollowDistanceCheck = null;
 
+        [Header("Events")]
+        /// <summary>
+        /// Event that gets fired when auto follow is triggered
+        /// </summary>
+        public UnityEvent AutoFollowTriggered = new UnityEvent();
```

Comment:
[We should make this a private field](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation) with the `SerializeField` tag, with a property accessor if needed

Reply:
Updated. Thanks!